interfaces/IUniswapAddressHolderAave.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.12;
+pragma abicoder v2;
+
+interface IUniswapAddressHolderAave {
```

Comment:
I don't like this name, maybe include the version of solidity in the name instead of "Aave"